pyhealth/metrics/ranking.py

```diff
@@ -0,0 +1,38 @@
+from typing import List, Dict
+
+
+def ranking_metrics_fn(qrels: Dict[str, Dict[str, int]],
```

Comment:
Could you add an example for this ranking function? How to use it, what is query, result, and kvalues?